src/pages/verify/e-ids/german-personalausweis.mdx

```diff
@@ -0,0 +1,127 @@
+        </tr>
+        <tr>
+            <td>Profile 1</td>
+            <td>Pseudonym, First Name, Family Name, Religious Order / Artistic Name, Academic Title, Date of Birth, Place of Birth, Birth Name</td>
+            <td>User registration/one-time authentication</td>
+        </tr>
+        <tr>
+            <td>Profile 2</td>
+            <td>Everything in Profile 1, plus Nationality and Address (normal place of residence)</td>
+            <td>Know-your-customer (KYC)</td>
+        </tr>
+        <tr>
+            <td>Profile 3</td>
+            <td>Everything in Profile 2, plus Document Type, Issuing Country (always "D" for Germany), Date of Expiry and Residence Permit I</td>
+            <td>Extended KYC with document metadata</td>
+        </tr>
+    </tbody>
+</table>
+
+For applications configured to use a `dynamic` `scope` strategy, a profile can be selected on each request using the `scope` parameter prefixed with `odis:`, e.g. `odis:Profile1`.
+For applications configured to use a `static` `scope` strategy, the profile will always be `Pseudonym`.
+
+It is possible to define your own profiles if you have special requirements; if you would like to do so, please contact sales.
+
+### Transaction information
+You can add a reference text to display to the user while they scan the physical Personalausweis with the AusweisApp.
+The reference text consists of a _template_ with a single placeholder and a _dynamic parameter_ called `txinfo`.
+If a template is registered for the client, `txinfo` _must_ be passed as a `login_hint` on each request.
+The template may be up to 118 characters long and can contain any characters.
+The `txinfo` parameter can be up to 32 characters long and can only contain alphanumeric characters (in particular, it cannot contain spaces).
+
+### Example (partial) test authorize requests
+Criipto provides a test client on `adesso-test.criipto.id`.
+You can use this client to test your integration before ordering your own client credentials (see below).
+The test client has a transaction information template registered, so you _must_ pass a `txinfo` parameter on every request.
+The template is set up such that you can for instance use your company name as the `txinfo` parameter.
+
+Here are some examples of partial authorize requests:
+```
+https://adesso-test.criipto.id/oauth2/authorize?scope=openid odis:Profile3&login_hint=txinfo:TestCompany&...
+```
+Alternatively, you can send the `scope` in the `login_hint`:
+```
+https://adesso-test.criipto.id/oauth2/authorize?...&login_hint=scope:odis:Profile1 txinfo:TestCompany&...
+```
+which can be a useful if you are working with technology that does not let you control the `scope` value.
+
+## Getting ready for production
+To start accepting real users with Personalausweis, you must first request your own _client credentials_ and provide a _client certificate_ to be signed.
+The credentials consists of a _client id_ and a _client secret_.
```

Comment:
The credentials consists => The credential consists

Reply:
Fixed in 00bfb36.